frontend/src/GuidanceTagList.js

```diff
@@ -66,12 +66,35 @@ export function GuidanceTagList({
       )
     })
 
-  const properlyConfigured = (
-    <Stack isInline align="center" bg="strongMuted" px="2">
-      <Icon name="check-circle" color="strong" />
-      <Text>
-        <Trans>Properly configured!</Trans>
-      </Text>
+  const smallDevice = window.matchMedia('(max-width: 500px)').matches
+  const noTags = (
+    <Stack isInline align="center" bg="moderateMuted" px="2">
+      {!smallDevice && <Icon name="warning-2" color="moderate" />}
+      <Box>
+        <Stack isInline align="center">
+          {smallDevice && <Icon name="warning-2" color="moderate" />}
+          <Text fontWeight="bold">
+            <Trans>Result:</Trans>
+          </Text>
+          <Text>
+            <Trans>No guidance tags were found for this scan category</Trans>
+          </Text>
+        </Stack>
+        <Stack isInline>
+          <Text fontWeight="bold">
+            <Trans>Guidance:</Trans>
+          </Text>
+          <Text>
+            <Trans>
+              This could be due to improper configuration, or could be the
+              result of a scan error
+            </Trans>
+          </Text>
+        </Stack>
+        <Text fontWeight="bold">
+          <Trans>Please use the contact form to notify an administrator</Trans>
```

Comment:
this needs to exist...

Reply:
Until a contact form exists with an official email, what about just saying "If you believe this was caused by a problem with Tracker, use the 'Report an Issue' link below"?

Reply:
yup, something liket hat works